tools/tests/bundle-ignore-files.js

```diff
@@ -0,0 +1,15 @@
+const selftest = require('../tool-testing/selftest.js');
+const bundler = require('../isobuild/bundler.js');
+
+selftest.define("bundle-ignore-files", () => {
+  const patterns = bundler.ignoreFiles;
+  const matchingInputs = [
+    '.git/',
+    '.meteor/',
+    '.DS_Store',
+    '.aaabbb.swp',
+    'Thumbs.db'
+  ];
+
+  matchingInputs.forEach(input => selftest.expectEqual(patterns.some(p => p.test(input)),true));
```

Comment:
Yeah, i meant to do that at first, but honestly, the regex is so simple it seemed a bit pointless.